README.md

```diff
@@ -165,6 +170,4 @@ If you like this project kind assist in giving a star
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] You still haven't fixed the MIT link. Kindly do that.

  ![image](https://github.com/Ademola-coding/Blog_rails/assets/36542986/ed67dd10-1fef-4eb9-af8f-1b0c1a6053f2)

Reply:
Done